src/fear_and_greed/cnn.py

```diff
@@ -62,3 +53,24 @@ def get(fetcher: Fetcher = None) -> FearGreedIndex:
         description=response["rating"],
         last_update=datetime.datetime.fromisoformat(response["timestamp"]),
     )
+
+def historical(fetcher: Fetcher = None) -> dict:
```

Comment:
We're going to need tests for this function. You can follow the example of those for `get()`.

Reply:
Thanks, for pointing that out, I did not see the tests